headers/types/dungeon_mode/dungeon_mode.h

```diff
@@ -138,11 +142,12 @@ struct statuses {
     undefined field_0x38;
     undefined field_0x39;
     undefined field_0x3a;
-    undefined field_0x3b;
-    undefined field_0x3c;
-    undefined field_0x3d;
-    undefined field_0x3e;
-    undefined field_0x3f;
+    // 0x3B: Leech seed related tracker. Set to the value in statuses:0x7 after finding
+    // statuses::leech_seed_source_monster_index.
+    uint32_t field_0x3b;
```

Comment:
Can we name this something with `leech_seed`? Like `unk_leech_seed_tracker` or something. If we have a general sense of what this is for, I would rather have a descriptive name because you don't really see comments in Ghidra's decompiler.